meerk40t/gui/panes/navigationpanels.py

```diff
@@ -957,12 +982,10 @@ def scaleit(self, scale):
         x = input_driver.current_x if input_driver is not None else 0
         y = input_driver.current_y if input_driver is not None else 0
         self.context(
-            "scale %f %f %f %f\n"
+            "scale %f %f \n"
             % (
                 scale,
-                scale,
-                x,
-                y,
+                scale
```

Comment:
Style-wise we should likely just apply black and call it a day.

As code thing it reduces bickering over style bits. Since it always transforms everything into the same thing anyway.